### ISSUE TITLE
Add documentation to log parser fields

### ISSUE THREAD

**Issue by rleighton** [panther-labs/panther#115]
Update all log parser fields to include a 'description' tag so that columns in Athena and Glue tables will show the field level documentation to users. For example, we have already done this with nginx logs. When you hover over a column in Athena you see the documentation (also we will use this in the UI to show users the definition of the fields):

![image](https://user-images.githubusercontent.com/18419762/73869393-2bd4c200-4818-11ea-9ed8-7b40078d9b73.png)

**Comment by rleighton** [comment]
Code complete, pending submitting PR.

**Comment by jacknagz** [comment]
Hey @rleighton would you mind adding some more context?

**Comment by rleighton** [comment]
Updated with screen shot of Athena. This is ready to be pushed , just waiting on the common fields PR to push, then this one.

**Comment by rleighton** [comment]
Done
